CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 
 ## [Unreleased]
 
+### Breaking Changes
+
+- when setting an IPv6 address using the [DIR|SD|FD] Addresses directive, now bareos only listens on IPv6 only instead of both IPv4 and IPv6 on dual-stack. If you were using the [DIR|SD|FD] Addresses directive to create a dual-stack socket that would listen on both IPv6 AND IPv4, it won't work that way anymore. You should now also explicitely specify the IPv4 address in the directive [PR #882]
```

Comment:
```suggestion
- when setting an IPv6 address using the [DIR|SD|FD] Addresses directive, now bareos only listens on IPv6 only instead of both IPv4 and IPv6 on dual-stack. If you were using the [DIR|SD|FD] Addresses directive to create a dual-stack socket that would listen on both IPv6 AND IPv4, it won't work that way anymore. You should now also explicitely specify the IPv4 address in the directive or remove the directive as listening on IPv6 and IPv4 is now the default. [PR #882]
```

Reply:
I hope we are clear, with good sample of configuration in documentation ...

Reply:
the last one is the new default, and the first one didn't work previously (we just fixed it a few minutes ago).
You can also do `Dir Address = 0.0.0.0` to get IPv4 only. I hope it is quite obvious the way it is implemented right now.